ChemistryLib/Output.h

```diff
@@ -0,0 +1,102 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#pragma once
+
+#include <algorithm>
+#include <fstream>
+#include <string>
+#include <vector>
+
+namespace ChemistryLib
+{
+class BasicOutputSetups
+{
+public:
+    static inline const std::string output_file = "phreeqc.out";
```

Comment:
Using the same input/output files does not go well when running tests in parallel. This will lead to strange errors because half of the inputs/outputs will be from another simulation. Using a random file name in the temporary dir is recommended.

Reply:
Changed as recommended.